console/console-init/ui/src/modules/address-space/components/AddressSpaceNavigation/AddressSpaceNavigation.tsx

```diff
@@ -46,6 +46,19 @@ export const AddressSpaceNavigation: React.FunctionComponent<AddressSpaceNavigat
             Connections
           </NavLink>
         </NavItem>
+        <NavItem
+          key="endpoints"
+          itemId="endpoints"
+          isActive={active === "endpoints"}
+        >
+          <NavLink
+            id="ad-space-nav-endpoints"
+            to={`endpoints`}
+            style={{ color: "black" }}
```

Comment:
use css class instead of inline css and use patterfly global variables for color.